connect-inject/container_init_test.go

```diff
@@ -91,6 +91,27 @@ func TestHandlerContainerInit(t *testing.T) {
 			"",
 			`datacenter`,
 		},
+		{
+			"Prepared Query specified",
+			func(pod *corev1.Pod) *corev1.Pod {
+				pod.Annotations[annotationService] = "web"
+				pod.Annotations[annotationUpstreams] = "prepared_query:handle:1234"
+				return pod
+			},
+			`destination_type = "prepared_query"`,
+			"",
```

Comment:
```suggestion
			`destination_type = "service"`,
```

Ooo, we could make this test do more work for us by checking that `destination_type = "service"` is not included as part of it!